website/templates/public/pages/faq.mako

```diff
@@ -20,7 +20,7 @@
             href="/getting-started">Getting
         Started</a> videos, see the <a
             href="/4znZP/wiki/home">OSF
-        Features </a>page,</p> or see how other scientists <a href="/svje2/">use the OSF.</a>
+        Features </a>page, or see how other scientists <a href="/svje2/">use the OSF.</a>
```

Comment:
Necessary is a little strong, but yes, I could move it to however I can move it to after the last a. We want to avoid that famine.